Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/service/impl/ConceptEditService.java

```diff
@@ -0,0 +1,74 @@
+package edu.asu.conceptpower.app.service.impl;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import edu.asu.conceptpower.app.bean.ConceptEditBean;
+import edu.asu.conceptpower.app.core.IConceptManager;
+import edu.asu.conceptpower.app.exceptions.IndexerRunningException;
+import edu.asu.conceptpower.app.exceptions.LuceneException;
+import edu.asu.conceptpower.app.service.IConceptEditService;
+import edu.asu.conceptpower.core.ConceptEntry;
+
+@Service
+public class ConceptEditService implements IConceptEditService {
+
+    @Autowired
+    private IConceptManager conceptManager;
+
+    /**
+     * This method adds all the wordnet concepts that are removed from wordnet
+     * id field of wrapper. The next step is to remove all the wordnet concepts
+     * that are added to the wordnet id field of the wrapper. Finally the
+     * concept is updated.
```

Comment:
what does updated mean here? saved in database, values are updated, index is updated?